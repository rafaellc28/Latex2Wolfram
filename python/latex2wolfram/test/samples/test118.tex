% Test Jordan decomposition
Jordan(\begin{bmatrix}
 -10 & 1 & 8\\
  -7 & 2 & 3\\
  16 & 2 & 12\\
\end{bmatrix})
