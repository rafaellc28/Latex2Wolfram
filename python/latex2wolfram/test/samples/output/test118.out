Jordan({{-10, 1, 8}, {-7, 2, 3}, {16, 2, 12}})
